docs/dev-guide.md

```diff
@@ -1,67 +1,28 @@
 
-- All course configuration is done via plain text files in the git repository, which is the master source for this data. There is no extra course configuration stored in the DB. The instructor does not directly edit course data via the website.
+- The majority of course content and configuration is done via plain text files in the git repository, which is the master source for this data.
 
 - All student data is all stored in the DB and is not pushed back into the git repository or disk at any point.
 
-## Directory layout
-
-- Broadly follow the [Express generator](http://expressjs.com/en/starter/generator.html) layout.
-
-- Top-level files and directories are:
-
-  ```text
-  PrairieLearn
-  +-- autograder         # files needed to autograde code on a separate server
-  |   `-- ...            # various scripts and docker images
-  +-- config.json        # server configuration file (optional)
-  +-- cron               # jobs to be periodically executed, one file per job
-  |   +-- index.js       # entry point for all cron jobs
-  |   `-- ...            # one JS file per cron job, executed by index.js
-  +-- docs               # documentation
-  +-- exampleCourse      # example content for a course
-  +-- lib                # miscellaneous helper code
-  +-- middlewares        # Express.js middleware, one per file
-  +-- migrations         # DB migrations
-  |   +-- ...            # one PGSQL file per migration, executed in order of their timestamp
-  +-- package.json       # JavaScript package manifest
-  +-- pages              # one sub-dir per web page
-  |   +-- partials       # EJS helper sub-templates
-  |   +-- instructorHome # all the code for the instructorHome page
-  |   +-- userHome       # all the code for the userHome page
-  |   `-- ...            # other "instructor" and "user" pages
-  +-- public             # all accessible without access control
-  |   +-- javascripts    # external packages only, no modifications
-  |   +-- localscripts   # all local site-wide JS
-  |   `-- stylesheets    # all CSS, both external and local
-  +-- question-servers   # one file per question type
-  +-- server.js          # top-level program
-  +-- sprocs             # DB stored procedures, one per file
-  |   +-- index.js       # entry point for all sproc initialization
-  |   `-- ...            # one JS file per sproc, executed by index.js
-  +-- sync               # code to load on-disk course config into DB
-  `-- tests              # unit and integration tests
-  ```
-
 ## Unit tests and integration tests
 
-- Tests are stored in the `tests/` directory and listed in `tests/index.js`.
+- Tests are stored in the `apps/prairielearn/src/tests/` directory..
```

Comment:
This likely needs to be split between unit tests and CI tests. Many unit tests have moved to `.test.ts` files in the location of the original function.

Reply:
Changed in 7ec3678.